pkg/bootstrap/docker/dockerhelper/helper.go

```diff
@@ -35,22 +33,27 @@ type RegistryConfig struct {
 	InsecureRegistryCIDRs []string
 }
 
-func getRegistryConfig(env *docker.Env) (*RegistryConfig, error) {
-	for _, entry := range *env {
-		if !strings.HasPrefix(entry, "RegistryConfig=") {
-			continue
+func getRegistryConfig(env *docker.DockerInfo) (*RegistryConfig, error) {
+	// TODO: Disabled because the latest go-dockerclient dropped docker.Env and
```

Comment:
@csrwng FYI (the client.Info() is now returning this and I was not able to find the 'env' or the RegistryConfig there.